db/migrate/20230202222246_create_jwt_denylist.rb

```diff
@@ -0,0 +1,9 @@
+class CreateJwtDenylist < ActiveRecord::Migration[7.0]
+  def change
```

Comment:
- [ ] Great job so far 👏🏻 👏🏻  However some error has occurred, while running the migration (rails db:migrate) this occurred because all later migrations were canceled. Kindly fix this 💯 Here is the screenshot of the error 👇🏻 

![image](https://user-images.githubusercontent.com/79058364/216487694-e02ec1f7-f65d-4f90-8128-c7d378c294cb.png)

Reply:
how can i fix?

Reply:
![image](https://user-images.githubusercontent.com/102005284/216489288-b1ead84a-f970-457d-bcd1-9cb0abc07f5a.png)
that is my migration status

Reply:
![image](https://user-images.githubusercontent.com/102005284/216489577-0ce41846-41c4-42cf-9910-faf0e32a8377.png)
and this is the table users

Reply:
@jdbs9514, Can you please clone the project once again then try to create a data base and run the Migration and if the app run without any error, Please submit for the rereview the next code review will check it out and if everything is okay your project will be approve 💯 👍🏻